statsmodels/genmod/tests/test_glm_weights.py

```diff
@@ -182,3 +221,367 @@ def setupClass(cls):
         # compare with discrete, start close to save time
         #modd = discrete.Poisson(cpunish_data.endog, cpunish_data.exog)
         self.res2 = res_stata.results_poisson_fweight_clu1
+
+
+class TestGlmTweedieAwNr(CheckWeight):
+    @classmethod
+    def setupClass(cls):
```

Comment:
rename to `setup_class`
we will switch to pytest in future which doesn't support the camelCase